hero-dashboard/shortcuts/hero-dashboard.desktop

```diff
@@ -13,7 +16,6 @@ Keywords=Run;
 Actions=New
 X-Ubuntu-Gettext-Domain=gnome-terminal
 Terminal=false
-Exec=bash -ic hero-dashboard
-Icon[en_US]=/usr/share/pixmaps/tue/hsr.png
-Name[en_US]=HERO Dashboard
-Name=HERO Dashboard
+Exec=bash -c "source ~/.tue/setup.bash && hero-dashboard"
```

Comment:
Have you tried
```suggestion
Exec=bash -c "source .bashrc && hero-dashboard"
```

Reply:
Doesn't work, as on top of `.bashrc`, it check if it interactive, if not, it returns